tmail-backend/tmail-third-party/openpaas/docker-compose-carddav-sample.yml

```diff
@@ -0,0 +1,132 @@
+version: "3"
+
+networks:
+  emaily:
+    driver: bridge
+
+services:
+  openpass:
```

Comment:
```suggestion
  openpaas:
```

Reply:
Cool. Can we add a bit README about this docker compose?

Reply:
I have already noted it in the `openpaas.adoc`. If I create a new README, I am unsure of what else to write.